src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -1,7 +1,16 @@
-import { Button, Dialog, Flex, FlexProps, color, media } from "@artsy/palette"
+import {
+  Button,
+  Dialog,
+  Flex,
+  FlexProps,
+  color,
+  media,
+  space,
+} from "@artsy/palette"
 import { Conversation_conversation } from "v2/__generated__/Conversation_conversation.graphql"
 import React, { useRef, useState } from "react"
 import { Environment, RelayRefetchProps } from "react-relay"
+import { unica } from "v2/Assets/Fonts"
```

Comment:
Tthis is super old legacy (pre palette) and shouldn't be used

Reply:
Wow, I will adjust it now :)